CHANGELOG.unreleased.md

```diff
@@ -39,6 +39,7 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- xo-server-audit minor
```

Comment:
Shouldn't it be `patch` instead?

Reply:
It's not a bug it's an improvement. The audit plugin works fine without this PR.